getting_started/step_by_step/your_first_game.rst

```diff
@@ -779,9 +779,13 @@ You can assign this property's value in two ways:
 - Click the down arrow next to "[empty]" and choose "Load". Select
   ``Mob.tscn``.
 
-Next, click on the Player and connect the ``hit`` signal. We want to make a
-new function named ``game_over``, which will handle what needs to happen when a
-game ends. Type "game_over" in the "Receiver Method" box at the bottom of the
+Next, click on the ``Player`` node and, if you have not, select the Node-tab
+instead of the Inspector-tab from the sidebar. You should see a list of the
+signals for the ``Player`` node. Find and double-click the ``hit`` signal in
+the list (or right-click it and select "Connect..."). This will open the
+"Connect a Signal" window. We want to make a new function named ``game_over``,
+which will handle what needs to happen when a game ends.
```

Comment:
I'd rephrase a bit, e.g.:
```suggestion
Next, select the ``Player`` node in the Scene dock, and access the Node dock on
the sidebar. Make sure to have the Signals tab selected in the Node dock.

You should see a list of the signals for the ``Player`` node. Find and
double-click the ``hit`` signal in the list (or right-click it and select
"Connect..."). This will open the the signal connection dialog. We want to make
a new function named ``game_over``, which will handle what needs to happen when
a game ends.
```

Reply:
skipped the double the and changed the second instance of `"Connect a Signal" window` into `signal connection dialog`